### DISCUSSION TITLE
如何在 Android 12L x86_64 上安装 LSPosed?

### DISCUSSION THREAD

**Discussion by 1457384613gh** [LSPosed/LSPosed#1937] [Q&A] [answered]
我想在 Android 12L x86_64 上面使用一些 Xposed 模块。
Android 12L x86_64 自带 kernel-SU 但是没有 Magisk ，所以无法常规方法安装 LSPosed。
https://sourceforge.net/projects/blissos-dev/files/Android-Generic/PC/aosp/gapps/12L/
可以像安装 XPosed 那样，解zip包，然后通过 root 权限推到 /system 中吗？
显然不行，需要 Riru 或者 zygisk ，如果 Riru 也一起解包，推送进 /system 呢？
有别的绕开 Magisk，并且借助 kernel-SU 实现安装 LSPosed 的办法吗？？？

**Comment by yujincheng08** [comment, marked as answer]
kernelSU 可以启动 magisk 啊：
https://github.com/topjohnwu/Magisk/blob/master/scripts/avd_magisk.sh

**Reply by 1457384613gh** [reply]
谢谢，我已经弄好了
